desktop/core/src/desktop/js/ko/components/ko.shareGistModal.js

```diff
@@ -83,7 +83,8 @@ class ShareGistModal extends DisposableComponent {
     try {
       await ApiHelper.sendSlackMessageAsync({
         channel: this.selectedChannel,
-        message: this.messageDescription() + '\n' + this.link()
+        message:
+          '@' + window.LOGGED_USERNAME + ': ' + this.messageDescription() + '\n' + this.link()
```

Comment:
This is super hackable and better to also try to give the name first instead of login.

Check `doc.owner.get_full_name() or doc.owner.username` in the backend?

Reply:
My bad, updated the message in the backend using `request.user`